src/Migrations/Connection/ConnectionFactory.php

```diff
@@ -44,6 +45,7 @@ public function createFakeConnection()
      */
     public function createConnection(array $configuration)
     {
+        Type::addType('unicodetext', 'Jadu\DoctrineTypes\UnicodeTextType');
```

Comment:
Not sure if PHP-CS-Fixer will put a new line before the return. Try running it if you haven't already just to be sure.